src/dashboard/src/components/api/views.py

```diff
@@ -80,18 +80,18 @@ def wrapper(request, *args, **kwargs):
 def authenticate_request(request):
     error = None
 
+    client_ip = request.META['REMOTE_ADDR']
+    whitelist = helpers.get_setting('api_whitelist', '').split()
+    if client_ip in whitelist:
+        LOGGER.debug('API called by trusted IP %s', client_ip)
+        return None
+
```

Comment:
I've added the migration (0033 in dashboard), finally decided to *always* reset the value to empty string. I think that's the safest.